P5/Source/Specs/person.xml

```diff
@@ -69,16 +69,15 @@ specification.</p></remarks>
       <desc versionDate="2007-05-04" xml:lang="es">especifica el sexo de una persona.</desc>
       <desc versionDate="2007-01-21" xml:lang="it">indica il sesso di una persona</desc>
       <datatype minOccurs="1" maxOccurs="unbounded"><dataRef key="teidata.sex"/></datatype>
-      <remarks versionDate="2013-04-14" xml:lang="en">
-        <p>Values for this attribute may be locally defined by a project, or may refer to an external standard, such as
-          vCard's sex property <ptr target="http://microformats.org/wiki/gender-formats"/>
-          (in which <val>M</val> indicates male, <val>F</val> female, <val>O</val> other, <val>N</val> none or not applicable, <val>U</val> unknown),
-          or the often used ISO 5218:2004 <title>Representation of Human Sexes</title>
-               <ptr target="http://standards.iso.org/ittf/PubliclyAvailableStandards/c036266_ISO_IEC_5218_2004(E_F).zip"/> (in which <val>0</val>
-          indicates unknown; <val>1</val> male; <val>2</val> female; and <val>9</val> not applicable,
-          although the ISO standard is widely considered inadequate);
-          cf. CETH's <title>Recommendations for Inclusive Data Collection of Trans People</title>
-               <ptr target="http://transhealth.ucsf.edu/trans?page=lib-data-collection"/>.</p>
+      <remarks versionDate="2022-05-18" xml:lang="en">
+        <p>Values for this attribute may be locally defined by a project or according to an external standard.</p>
```

Comment:
“Values for this attribute may be locally defined by a project, or may refer to an external standard.”
Or the wording in comment above.